media/js/src/main.js

```diff
@@ -8,8 +8,30 @@ requirejs.config({
 });
 
 define(['dragondrop'], function(DragonDrop) {
-    const demo1 = document.getElementById('dragondrop-container');
-    const dragonDrop = new DragonDrop(demo1, {
+    const el = document.getElementById('dragondrop-container');
+
+    const token = $('meta[name="csrf-token"]').attr('content');
```

Comment:
As these three variables are used only within `post_order`, I'd suggest moving them to within the function scope.

Reply:
@sdreher got it!